src/middlewared/middlewared/plugins/system.py

```diff
@@ -373,7 +373,7 @@ def ready(self):
         if delay:
             await asyncio.sleep(delay)
 
-        await Popen(["/sbin/poweroff"])
+        await Popen(['/sbin/poweroff'])
```

Comment:
We probably need to switch to `shutdown` here too?

Reply:
So `poweroff` equals to `shutdown -p now`. We are safe here :)